homeassistant/components/ifttt/__init__.py

```diff
@@ -17,6 +17,7 @@
 EVENT_RECEIVED = 'ifttt_webhook_received'
 
 ATTR_EVENT = 'event'
+ATTR_TO = 'to'
```

Comment:
We use `ATTR_TARGET` for notify platforms, I don't know if it would make sense to reuse here?